### ISSUE TITLE
Tooltips differing from chart info?

### ISSUE THREAD

**Issue by shelbyelavsky** [Chartability/POUR-CAF#8]
There are times when a chart's tooltip gives different information than what is being presented (e.g. a bar chart visually shows "group" and "value" text, but the tooltip lists "group, location, value, date.") 

What would be the best section to put this under? Technically, the metrics are defined, but only in a certain interaction. Would a new section added to Chartability be a better fit?

**Comment by frankelavsky** [comment]
Yeah, tooltips are so common in data visualizations that it would make sense to have something specific to this.

I'm thinking that we wouldn't want to hide essential information behind an interaction, or have a slightly more generalizable heuristic for this.

The REAL trick is digging up research or standards that clearly outline this, so we can point to them in our resources. We don't want to add a new heuristic without something solid to reference.

**Comment by frankelavsky** [comment]
## Issuing a call for help finding:
Standards or research resources that could help us argue that we shouldn't hide essential or different information in a tooltip.
